README.md

```diff
@@ -1 +1,72 @@
-# To-Do-list-list-structure
\ No newline at end of file
+# To Do List
+
+In this project, I built a simple HTML list of To Do tasks. This simple web page will be built using webpack and served by a webpack dev server.
+
+
+## Built With
+
+- HTMl
+- Webpack
+- CSS
+- Javascript
+
+## Live Demo (if available)
+
+## Getting Started
+
+1 - Above the list of files, click on the "Code" button
+
+2 - Copy the URL for the repository.
+
+3 - To clone the repository using HTTPS, under "HTTPS", click . To clone the repository using an SSH key, including a certificate issued by your organization's SSH certificate authority, click SSH, then click . To clone a repository using GitHub CLI, click GitHub CLI, then click . The clipboard icon for copying the URL to clone a repository with GitHub CLI
+
+4 - Open Git Bash.
+
+5 - Change the current working directory to the location where you want the cloned directory.
+
+6 - Type git clone, and then paste the URL you copied earlier.
+
+7 - Press Enter to create your local clone.
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Installation of Node.js, Node Package Manager, webpack module bundler
+Functioning computer Internet connection Basic understanding of Git and GitHub.
+
+### Setup
+[The setup of webpack](https://webpack.js.org/guides/getting-started/)
+
+### Install
+
+- A code editor (e.g. VSC) /text editor (e.g. notepad++)
+
+- Node.js
+
+## Authors
+
+👤 **Marcos Gualtero Lourenzo**
+
+- GitHub: [https://github.com/Goruchie]
+- Twitter: [https://twitter.com/Goruchie2]
+- LinkedIn: [https://www.linkedin.com/in/marcos-gualtero-a2aa35246/]
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it.